apps/sparo-lib/src/services/GitService.ts

```diff
@@ -390,6 +390,13 @@ Please specify a directory on the command line
     return Boolean(result);
   }
 
+  public isCommitSHA(shaMaybe: string): boolean {
+    const catFileResult: child_process.SpawnSyncReturns<string> = this.executeGitCommand({
+      args: ['cat-file', '-t', shaMaybe]
```

Comment:
`-t` returns the type of the object, which I think can be something like `blob`, `commit`, `tree`, `tag`.  Since any type is accepted, is  `isCommitSHA()` the best name?